tv_apps/smart-home/index.html

```diff
@@ -130,6 +132,12 @@
           </smart-button>
         </menu-group>
       </section>
+      <section id="search-bar" class="search-bar hidden">
+        <smart-button data-icon="mic" class="search-mic-button fade-out"></smart-button>
+        <div data-icon="search" class="search-input">
+          <span data-l10n-id="search-default-text">search</span>
```

Comment:
Of course, but it can help us to test if we make a typo of l10n id or attribute name.